assets/js/base/components/cart-checkout/totals/coupon/index.tsx

```diff
@@ -63,30 +68,33 @@ export const TotalsCoupon = ( {
 		if ( currentIsLoading.current !== isLoading ) {
 			if ( ! isLoading && couponValue && ! validationError ) {
 				setCouponValue( '' );
+				setCouponFormClass( 'screen-reader-text' );
 			}
 			currentIsLoading.current = isLoading;
 		}
 	}, [ isLoading, couponValue, validationError ] );
 
 	return (
-		<Panel
-			className="wc-block-components-totals-coupon"
```

Comment:
removing this class is a regression. We still need to keep the class in the DOM. The `<>` below should have this class.